NSS-Spontaneous-Spoonbills/Sprint2
# Em computerview

## Description of Proposed Changes

* Added Computer List view
* Made minor changes to index.html template
* Created empty template files for my upcoming views

## Steps to Test

```sh
git fetch --all
git checkout em_computerview
```

1. Start your virtual environment and your server
1. Run the following in the command line: 

```shell
python manage.py makemigrations WorkforceManagement
python manage.py migrate
```
1. In DB Browser, create 1-2 computers in the "Computers" table
1. Navigate to ```http://127.0.0.1:8000/WorkforceManagement/computers/```
1. Verify that the computers you created are displayed. 


## Impacted Areas in Application

**New Files**
WorkforceManagement/templates/WorkforceManagement/Computer_Detail.html
WorkforceManagement/templates/WorkforceManagement/Computer_List.html
WorkforceManagement/templates/WorkforceManagement/Computer_New.html
WorkforceManagement/templates/WorkforceManagement/Computer_Update.html

**Updated Files**
README.md
WorkforceManagement/models/Computer_Model.py
WorkforceManagement/models/__init__.py
WorkforceManagement/templates/WorkforceManagement/index.html
WorkforceManagement/urls.py
WorkforceManagement/views/Computer_View.py
WorkforceManagement/views/__init__.py


## Mentions @username

@JacobAndrewSmith92 @binauralbeat @obscure-shadow 

## Testing
